SWLOR.Game.Server/Service/DurabilityService.cs

```diff
@@ -163,11 +165,11 @@ public void RunItemDecay(NWPlayer player, NWItem item, float reduceAmount)
             
             float durability = GetDurability(item);
             string sItemName = item.Name;
-            
+            int apr = _creature.GetAttacksPerRound(player, 0);
```

Comment:
I'm not familiar with this NWNX function but based on the comment I think this should be set to true, not false. We want to take into account all sources of BAB, not just the ones set via the SetBaseAttackBonus method.

As an ancillary problem, the function should take a bool not an int. Not a problem you created - I just didn't do it when I converted it from NWScript. I'd appreciate it if you'd fix that as part of this.